content/html/reference/elements/acroynm/acronym.md

```diff
@@ -0,0 +1,44 @@
+---
+recipe: html-element
```

Comment:
I like having `recipe` first but most of our files have it last. We should consider having a common order (maybe just: the recipe comes first, then items are the same order as in the recipe?).

Reply:
I like the idea of having the recipe first and deriving the order from the recipe _a lot_. Fixing these automatically could be very amenable to scripting. Like prettier or other formatters, you could run it on save, so you don't even have to think about it, at least as an author.

Reply:
Opened #212